action.yaml

```diff
@@ -71,7 +71,7 @@ runs:
         if [ $(echo $PREFECT_SHORT_VERSION'<'10.14 | bc -l) -eq 1 ];
         then
           echo "The currently installed Prefect version (2.$PREFECT_SHORT_VERSION) will be updated to the latest to ensure compatibility with this action."
-          pip install prefect -U "prefect>=2.10.14"
+          pip install prefect -U "prefect>=2.10.14,<3"
```

Comment:
quick q why did we make the floor 2.10.14?

Reply:
I believe this is when the deploy cli command was added

Reply:
good point